api-reference/beta/resources/application.md

```diff
@@ -18,82 +18,70 @@ Here is a JSON representation of the resource
-|samlMetadataUrl|String|The URL to the SAML metadata for the application.|
+|tags|String collection| |
+|web|[web](web.md)| Specifies settings for a web application. |
 
 ### Relationships
-| Relationship | Type |Description|
+
+| Relationship | Type | Description |
 |:---------------|:--------|:----------|
-|connectorGroup|[connectorGroup](connectorgroup.md)| The connectorGroup the applicaiton is using with Azure AD Application Proxy. Nullable.| 
+|connectorGroup|[connectorGroup](connectorgroup.md)| The connectorGroup the application is using with Azure AD Application Proxy. Nullable.|
 |createdOnBehalfOf|[directoryObject](directoryobject.md)| Read-only.|
 |extensionProperties|[extensionProperty](extensionproperty.md) collection|The extension properties associated with the application. Read-only. Nullable.|
 |owners|[directoryObject](directoryobject.md) collection|Directory objects that are owners of the application. The owners are a set of non-admin users who are allowed to modify this object. Requires version 2013-11-08 or newer.  Read-only. Nullable.|
 |policy|[policy](policy.md) collection|The policies assigned to this application.|
 
 ### Methods
 
-| Method       | Return Type  |Description|
+| Method | Return Type | Description |
 |:---------------|:--------|:----------|
 |[Get application](../api/application_get.md) | [application](application.md) |Read properties and relationships of application object.|
 |[Create extensionProperty](../api/application_post_extensionproperties.md) |[extensionProperty](extensionproperty.md)| Create a new extensionProperty by posting to the extensionProperties collection.|
 |[List extensionProperties](../api/application_list_extensionproperties.md) |[extensionProperty](extensionproperty.md) collection| Get a extensionProperty object collection.|
 |[List assigned policies](../api/policy_list_assigned.md)| [policy](policy.md) collection| Get all policies assigned to this object.|
 |[Create owner](../api/application_post_owners.md) |[directoryObject](directoryobject.md)| Create a new owner by posting to the owners collection.|
-|[List owners](../api/application_list_owners.md) |[directoryObject](directoryobject.md) collection| Get a owner object collection.|
+|[List owners](../api/application_list_owners.md) |[directoryObject](directoryobject.md) collection| Get an owner object collection.|
 |[Update](../api/application_update.md) | [application](application.md) |Update application object. |
 |[Delete](../api/application_delete.md) | None |Delete application object. |
-|[restore](../api/application_restore.md)|[application](application.md)|
+|[Restore](../api/application_restore.md)|[application](application.md)|
```

Comment:
Remove.  We should be removing restore from the Application entity.  Restore is bound to directoryObjects, but I think the restore is done on a deletedItem.